EnglishFilter/sections/antiadblock.txt

```diff
@@ -2316,8 +2316,14 @@ gomovies.ec#%#window.check_adblock = true;
 ! https://github.com/AdguardTeam/AdguardFilters/issues/40813
 ! https://github.com/AdguardTeam/AdguardFilters/issues/38170
 ! https://forum.adguard.com/index.php?threads/30210/
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$empty,important,domain=unlockapk.com|newsjizz.in
+! https://github.com/AdguardTeam/AdguardFilters/issues/41895
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$empty,important,domain=newsjizz.in
```

Comment:
On my end with Chrome + AdGuard extension/app the rule with `$empty` modifier is enough, but if it's neccesary to unblock this request on your end then okay.

By the way, if you click on "download" button, it redirects to `newsjizz.in` and this website is using the same anti-adblock, so please add also `newsjizz.in` to the rules below and if needed to the rule above too.